Makefile

```diff
@@ -67,6 +69,14 @@ define error_handler
 	fi
 endef
 
+# Check if number of IR nodes from .env file is correct
+.PHONY: check_nodes
+check_nodes:
+	@if ! echo "$(AVAILABLE_NUMBER_OF_NODES)" | grep -wq "$(IR_NUMBER_OF_NODES)"; then \
+		echo "Error: IR_NUMBER_OF_NODES=$(IR_NUMBER_OF_NODES) is not in AVAILABLE_NUMBER_OF_NODES ($(AVAILABLE_NUMBER_OF_NODES))"; \
```

Comment:
can this error line be a little bit simpler? like "invalid IR number X; supported numbers: Y, Z, ..."